_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
```

Comment:
i think its time to factor that into a `_format_expl(lines, verbose)` function and invoke it before fixing the rewrite mod changes